src/openforms/js/components/admin/form_design/logic/actions/dmn/DMNActionConfig.js

```diff
@@ -0,0 +1,232 @@
+import {Form, Formik, useFormikContext} from 'formik';
+import PropTypes from 'prop-types';
+import React, {useContext, useState} from 'react';
+import {FormattedMessage, defineMessage} from 'react-intl';
+import {useAsync} from 'react-use';
+
+import {FormContext} from 'components/admin/form_design/Context';
+import {
+  DMN_DECISION_DEFINITIONS_LIST,
+  DMN_DECISION_DEFINITIONS_VERSIONS_LIST,
+} from 'components/admin/form_design/constants';
+import Field from 'components/admin/forms/Field';
+import Select from 'components/admin/forms/Select';
+import {get} from 'utils/fetch';
+
+import VariableMapping from './VariableMapping';
+import {inputValuesType} from './types';
+
+const ERRORS = {
+  required: defineMessage({
+    description: 'Required error message',
+    defaultMessage: 'This field is required.',
+  }),
+};
+
+const DecisionDefinitionIdField = () => {
+  const {values, setFieldValue, getFieldProps, errors, touched} = useFormikContext();
```

Comment:
Please consider using `useField` from formik, see remark below for the other field.